core/src/main/scala/kafka/server/ControllerServer.scala

```diff
@@ -131,11 +131,11 @@ class ControllerServer(
     if (!maybeChangeStatus(SHUTDOWN, STARTING)) return
     val startupDeadline = Deadline.fromDelay(time, config.serverMaxStartupTimeMs, TimeUnit.MILLISECONDS)
     try {
+      this.logIdent = new LogContext(s"[ControllerServer ${config.nodeId}] ").logPrefix()
       info("Starting controller")
       config.dynamicConfig.initialize(zkClientOpt = None)
 
       maybeChangeStatus(STARTING, STARTED)
-      this.logIdent = new LogContext(s"[ControllerServer id=${config.nodeId}] ").logPrefix()
```

Comment:
I think the idea of the `LogContext` stuff was to use key/value pairs so it's easy to enrich the log context. cc @jason for additional thoughts.

Reply:
Yes, this use-case is a bit of a hack. (A hack that this PR didn't add!)

Basically we're forced to use the old Scala `Logging` trait unless we want to rewrite all the BrokerServer stuff. But we want a nice-looking log prefix.